opentelemetry-proto/src/transform/logs.rs

```diff
@@ -98,6 +98,7 @@ pub mod tonic {
                         .map(Attributes::from_iter)
                         .unwrap_or_default()
                         .0;
+                    #[cfg(feature = "populate-logs-event-name")]
```

Comment:
should we do similar to consoleexporter?

Reply:
Yes done.